### DISCUSSION TITLE
How to enable sibling (implied relations)

### DISCUSSION THREAD

**Discussion by Dennyyusuf** [SkepticMystic/breadcrumbs#559] [Q&A]
Hello,

I am not sure whether this is a bug or not, but I can’t see the breadcrumb generate sibling relationships for the same parent relation.

I have three notes with the same author. In Excalibrain, it identifies that this up note has its siblings, but in the breadcrumb matrix view, it does not identify any sibling relationships.
<img width="1270" alt="image" src="https://github.com/SkepticMystic/breadcrumbs/assets/65077044/5f1f7a1c-66d0-4e08-a59c-aeb34844f24c">

Should I detail this sibling relationship in this setting?
<img width="652" alt="image" src="https://github.com/SkepticMystic/breadcrumbs/assets/65077044/c5d9c94d-6312-4234-94be-7b51a5483af2">

**Comment by csss1234** [comment]
I’ve only been obsessed with this plugin for about 2 days 😊 , but I think you need to change the “rounds” in your implied relation to `2`. It’s explained quite nicely here https://publish.obsidian.md/breadcrumbs-docs/Implied+Edge+Builders/Implied+Relation+Rounds
